test/parallel_api/iterator/iterators.pass.cpp

```diff
@@ -239,8 +248,16 @@ struct test_transform_iterator {
         test_transform_effect(in1.begin(),  in1.end(),  in2.begin());
         test_transform_effect(in1.cbegin(), in1.cend(), in2.begin());
 
-        auto new_transform_iterator = oneapi::dpl::make_transform_iterator(in2.begin(), [](T2& x) { return x + 1; });
-        test_random_iterator(new_transform_iterator);
+        transform_functor new_functor;
+        oneapi::dpl::transform_iterator<typename ::std::vector<T1>::iterator, transform_functor> _it1(in1.begin());
```

Comment:
`_it1` is unused variable here. You can use it, for example, for `base()` method testing or in another way

Reply:
Changed